dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -48,6 +48,9 @@ RUN pip3 install requests
 # We install the libpci module in order to be able to do PCI transactions
 RUN pip3 install libpci
 
+# Install psutil for process and system monitoring operations
+RUN pip3 install psutil==5.9.8
```

Comment:
I don't think it needs to be pinned to 5.9.8?

Reply:
Fixed in latest